pkg/shares/non_interactive_defaults.go

```diff
@@ -83,12 +86,43 @@ func roundUpBy(cursor, v int) int {
 	}
 }
 
-// MinSquareSize returns the minimum square size that can contain shareCount
+// BlobMinSquareSize returns the minimum square size that can contain shareCount
 // number of shares.
-func MinSquareSize(shareCount int) int {
+func BlobMinSquareSize(shareCount int) int {
```

Comment:
Do we want to write a couple of unit tests for this. I don't see any here

Reply:
ahh I forgot to rename the existing ones, this is the same func we've been using since ADR009, so those unit tests are still there I believe. Do these suffice?

https://github.com/celestiaorg/celestia-app/pull/1604/commits/0b2802e24c529e189ec743322139ec925be1e4e0

https://github.com/celestiaorg/celestia-app/blob/74c4b248a430183cef85ed0de6d06e9c37b2b255/pkg/shares/non_interactive_defaults_test.go#L372-L417